library/helpers/safeDecodeURIComponent.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as t from "tap";
+import { safeDecodeURIComponent } from "./safeDecodeURIComponent";
+
+t.test("it decodes a URI component", async (t) => {
+  t.equal(decodeURIComponent("%20"), " ");
+  t.equal(decodeURIComponent("%3A"), ":");
+  t.equal(decodeURIComponent("%2F"), "/");
+  t.equal(decodeURIComponent("test%20test"), "test test");
+  t.equal(decodeURIComponent("test%3Atest"), "test:test");
```

Comment:
This is a native function, should we even test it? :D

Reply:
Needs to be `safeDecodeURIComponent`? I assume?

Reply:
Yep 😄. Also in case we wan't to use our own implementation.